src/components/Layout/Tabs.tsx

```diff
@@ -18,7 +18,7 @@ export type TabDefinition = {
   name: string,
   props?: any
 }
-type TabsLayout = "full-width" | "fixed-size" | "subtabs"
+type TabsLayout = "full-width" | "fixed-size" | "subtabs" | "subtabs-vertical"
```

Comment:
the new type `subtabs-vertical` creates the new style of tabs with a vertical separator

Reply:
👍